site/content/docs/5.2/forms/overview.md

```diff
@@ -164,6 +164,33 @@ Use the `.is-required` class on `label` to add a `*` symbol after it.
 </form>
 {{< /example >}}
 
+## Form helper
+
+Form helper is a button using `.form-helper` that provides extra help information to a form control. Be sure to include text for screen readers with a visually hidden text.
+
+On top of this button, apply a [popover]({{< docsref "/components/popovers" >}}) to display the help. A [tooltip]({{< docsref "/components/tooltips" >}}) can also be applied for small and simple text helpers without any actionable element inside.
+
+The helper button should be displayed right after the label.
+
+{{< example >}}
+<div class="mb-3">
+  <label for="tooltipTextInput" class="form-label">Input with helper inside a Popover</label>
+  <button type="button" class="form-helper" data-bs-toggle="popover" data-bs-placement="top" data-bs-content="Help for input">
```

Comment:
In terms of accessibility, there's no need to "link" this button that provides extra information to the associated label and/or input?

Reply:
FYI, a11y said that there's no need to add some aria selectors in here. Just the behavior of the tooltips/popovers need to be checked and changed here.